extension.Rmd

```diff
@@ -0,0 +1,241 @@
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred_prob, actual = actual, type = "rect", heading = "Model2: The Parsons Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 2](figures/SP.model2.png)
+
+#Conclusion:
+Again, there is little variation in our predictive probabilities with this model. Similar to model 1, most events are aligned with higher predictive probabilities on the right hand of the graph. Yet Model 1 has slightly better predictive power as the very end of the graph is likely to be clustered with events rather than non-events as in Model 2. 
+
+# Model 3: Group Conflict Model (Various theories)
+
+Group conflict theories encompass a variety of theories.  Some emphasize the role of instrumental rationality, arguing that xenophobia results from a contest between immigrants and residents over scarce resources like jobs and welfare benefits. Others emphasize the scapegoating of immigrants as perceived threats to residents' status, power or resources.  In this last model, we use relevant variables representing both the availability of resources like jobs and welfare benefits and the prevalence of immigration: unemployment rates, unemployment benefits, number of asylum seekers, saliance and variance of right-wing issues.
+
+We used interactions between contextual variables in this model in line with the author's original model.
+
+```{r}
+model_3 <- glm(ext_vote ~ z_asylumseeker + z_unemp + z_asylumseeker:z_unemp + z_replacement + z_replacement:z_unemp + z_replacement:z_asylumseeker + country - 1, data = data)
+
+summary(model_3)
+actual <- model_3$y # Create vector with actual values for vote intention for extreme right parties
+
+log.odds <- model_3$fitted.values # Create a function with log-odds
+pred_prob <- link(log.odds) # Transform log-odds into predicted probabilities
+pred_prob <- as.vector(pred_prob) 
+range(pred_prob) # variation in the predicted probabilities: very little variation.
+# Visualize variation in predicted probabilities
+pred_df <- data.frame(pred_prob)
+ggplot(pred_df, aes(x = 1:length(pred_prob), y = pred_prob)) + 
+  geom_line() +
+  xlab("Prediction No") +
+  ylab("Predicted Probability") +
+  ggtitle("Model 3: Variation in Predicted Probabilities")
+# Create separation plot to visualize the predictive power of the model
+separationplot(pred = pred_prob, actual = actual, type = "rect", heading = "Model3: The Group Conflict Thesis", xlab = "ER Vote Intention", newplot = F, line = T, lwd2 = 0.8, height = 0.5, show.expected = T) 
+```
+
+![Model 3](figures/SP.model3.png)
+
+#Conclusion:
+The majority of events (the intention to vote for an extreme right party) is clustered at the right end of the model - which is in line with a good predictive model. However, in this model the distribution of events is very dispersed compared to the other two models.
```

Comment:
very dispersed?  predicted probabilities seem to have less variation than the other models